src/main/java/com/automationexercise/Pages/TestCasesPage.java

```diff
@@ -12,13 +12,12 @@
 
 public class TestCasesPage {
     private final WebDriver driver;
-    private final By verification_ele =  By.xpath("//*[@id=\"form\"]/div/div[2]/h5/span");
-
+    private final By verifyElementVisible =  By.className("row");
```

Comment:
invalid locator: 
![image](https://github.com/Ashraaf7/AutomationExercisePracticing/assets/69063398/7cf9349d-ee61-4deb-8cc1-18c0a3734c83)
Try to get the Test Cases Label itself.